nodes/nodes-pods-priority.adoc

```diff
@@ -0,0 +1,43 @@
+// For example:
+// * file name: assembly_my-assembly-a.adoc
+// * ID: [id='assembly_my-assembly-a_{context}']
+// * Title: = My assembly A
+//
+// The ID is used as an anchor for linking to the module.
+// Avoid changing it after the module has been published
+// to ensure existing links are not broken.
+//
+// In order for  the assembly to be reusable in other assemblies in a guide,
+// include {context} in the ID: [id='a-collection-of-modules_{context}'].
+//
+// If the assembly covers a task, start the title with a verb in the gerund
+// form, such as Creating or Configuring.
+// The `context` attribute enables module reuse. Every module's ID
+// includes {context}, which ensures that the module has a unique ID even if
+// it is reused multiple times in a guide.
+:context: nodes-pods-priority
+[id='nodes-pods-priority_{context}']
+= Including pod priority in pod scheduling decisions in {product-title}
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+
+You can enable pod priority and preemption in your cluster. Pod priority indicates the importance of a pod relative to other pods and queues the pods based on that priority. Pod preemption allows the cluster to evict, or preempt, lower-priority pods so that higher-priority pods can be scheduled if there is no available space on a suitable node
+Pod priority also affects the scheduling order of pods and out-of-resource eviction ordering on the node. 
+
+To use priority and preemption, you create priority classes that define the relative weight of your pods. Then, reference a priority class in the pod specification to apply that weight for scheduling.
+
+Preemption is controlled by the `disablePreemption` parameter in the scheduler configuration file, which is set to `false` by default.
+
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
+
+include::modules/nodes-pods-priority-about.adoc[leveloffset=+2]
+include::modules/nodes-pods-priority-preempt-about.adoc[leveloffset=+2]
+include::modules/nodes-pods-priority-configuring.adoc[leveloffset=+2]
+include::modules/nodes-pods-priority-disabling.adoc[leveloffset=+2]
```

Comment:
Do all of these belong in the same user flow?

Reply:
I don't know, to be honest. That was how I structured things in the user story spreadsheet. Hoping a SME review can confirm/correct.

Reply:
That's totally fair - I'll be glad when we get those E2E reviews.